src/easy_config/__init__.py

```diff
@@ -117,6 +128,8 @@ def _read_file(
                 values[field.name] = value
             except (configparser.NoSectionError, configparser.NoOptionError):
                 pass
+            except (TypeError, ValueError) as e:
+                raise ConfigValueCoercionError(f'While reading the configuration file `{config_file if given_path else "UNKNWON"}`, could not coerce value for field `{field.name}` to type `{field.type}`') from e
```

Comment:
typo in UNKNOWN

Reply:
Does `field.type` always show something meaningful?

Reply:
Yeah, the dataclass decorator defines the `type` property on all the fields. If it's not something meaningful, something else is broken.